Security/Firewall/GetJWTListener.php

```diff
@@ -98,17 +99,27 @@ public function handle(GetResponseEvent $event)
     {
         $request = $event->getRequest();
 
-        if ($this->options['post_only'] && !$request->isMethod('POST')) {
-            $event->setResponse(new JsonResponse('invalid method', 405));
-            return;
+        $is_json_request = $request->headers->get('content_type') === 'application/json';
```

Comment:
Variables should use camelCase according to [Symfony CS](http://symfony.com/doc/current/contributing/code/standards.html#naming-conventions).